server.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const helmet = require('helmet');
+
+const db = require("./data/dbConfig");
+
+const server = express();
+
+const actionsRouter = require("./data/routers/actionsRouter");
+const projectsRouter = require("./data/routers/projectsRouter");
+
+server.use(helmet())
+server.use(express.json());
+// server.use(logger);
+server.use("/actions", actionsRouter);
+server.use("/projects", projectsRouter);
+
+server.get("/", async (req, res) => {
+    try {
+        const info = await db('info');
```

Comment:
If you notice the reason that this this did not work was because you were not queiring data only waiting to see the database was there